packages/tools/src/utilities/math/polyline/isPointInsidePolyline.ts

```diff
@@ -0,0 +1,98 @@
+import type { Types } from '@cornerstonejs/core';
+
```

Comment:
Please add unit tests for this function - particularly with directly on the line tests.